### DISCUSSION TITLE
Can't create Anchor on Issues

### DISCUSSION THREAD

**Discussion by Thao0907** [community/community#57186] [Projects and Issues]
### Select Topic Area

General

### Select Feature Area

- [X] Issues
- [ ] Projects

### Body

Hi everyone,

I make an Outline guide and I would like to have anchor ahead of each section so we can click on an Outline topic and it jumps directly to that section below.
It works on ".md" but when I create new issue, it doesn't.
Can anyone advise please ?
Thanks.

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to the GitHub feature areas [Issues](https://docs.github.com/en/issues/tracking-your-work-with-issues/about-issues) and/or [Projects](https://docs.github.com/en/issues/planning-and-tracking-with-projects/learning-about-projects/about-projects).

**Comment by aahouzi** [comment]
If you're creating an outline guide using Markdown and you want to include anchor links that allow users to jump to specific sections, it should work in Markdown files, including README.md files in repositories. However, it's important to note that GitHub issues do not support anchor links by default.

GitHub issues use a different rendering engine compared to Markdown files in repositories, which may not support all Markdown features, including anchor links. Therefore, when you create a new issue, the anchor links you've added will not function as expected.

If you need to include anchor links in a document that you want to share within GitHub issues, you have a couple of options:

1. Use a workaround with relative links: Instead of anchor links, you can use relative links to create a table of contents in your document. Each link can point to the corresponding section below, allowing users to navigate within the document. 

2. Share the Markdown file externally: If you need to include anchor links and have them functional, you can host the Markdown file externally and share the link to that file in your GitHub issue. There are various platforms and services where you can host Markdown files, such as GitHub Pages, GitLab Pages, or other web hosting services. This way, the anchor links should work as expected.

Please note that both options may require additional steps and may not provide the same seamless experience as anchor links in Markdown files within repositories. Choose the option that best suits your needs and consider the trade-offs.
